vlib/v/fmt/tests/struct_init_with_comments_keep.vv

```diff
@@ -13,8 +13,8 @@ pub fn new_builder(initial_size int) Builder {
 	return Builder{
 		// buf: make(0, initial_size)
 		buf:          []u8{cap: initial_size}
-		str_calls:    0 // after str_calls
-		len:          0 // after len
+		str_calls:    0            // after str_calls
+		len:          0            // after len
```

Comment:
Why should the *comments* be aligned too now?

Reply:
For the same reason as aligning the values...

Basically, it is often easier to read/comprehend the code if the text for such declarations is in a more "table" oriented layout, than jumbled all together.

Reply:
Because Go's auto formatting tool does it?

Reply:
It does 😞 : 
![image](https://github.com/user-attachments/assets/47696a57-d9a0-4b01-af59-ae62942e97f0)

Reply:
Go's fmt tool however, also keeps the newlines between the fields, and they act as separators, between the alignment blocks:
![image](https://github.com/user-attachments/assets/224daaf9-de46-458c-8d57-825043180012)

Reply:
The blank lines act as a separator between the "tables"... still easier to read than all jumbled with no common spacing.

Reply:
If I try to do the same in V:
```v
module abcde

pub struct Builder {
pub mut:
    // inline before field
    buf          []u8
    str_calls    int
    len          int
    initial_size int = 1
}

pub fn new_builder(initial_size int) Builder {
    return Builder{
        // buf: make(0, initial_size)
        buf:          []u8{cap: initial_size}
        str_calls:    0            // after str_calls
        len:          0            // after len

        initial_size: initial_size // final
    }
}
```
that gets formatted first to (note how the comments are ok, but the separator newline was removed):
```v
module abcde

pub struct Builder {
pub mut:
	// inline before field
	buf          []u8
	str_calls    int
	len          int
	initial_size int = 1
}

pub fn new_builder(initial_size int) Builder {
	return Builder{
		// buf: make(0, initial_size)
		buf:       []u8{cap: initial_size}
		str_calls: 0 // after str_calls
		len:       0 // after len
		initial_size: initial_size // final
	}
}
```

... and then, since the separator line is removed, the output is no longer invariant, and I have to run `v fmt -w ./vlib/v/fmt/tests/struct_init_with_comments_keep.vv` a second time, getting this time to:
```v
module abcde

pub struct Builder {
pub mut:
	// inline before field
	buf          []u8
	str_calls    int
	len          int
	initial_size int = 1
}

pub fn new_builder(initial_size int) Builder {
	return Builder{
		// buf: make(0, initial_size)
		buf:          []u8{cap: initial_size}
		str_calls:    0            // after str_calls
		len:          0            // after len
		initial_size: initial_size // final
	}
}
```

V fmt *should not have to be run twice* (the output should be stable), and it should keep the single newlines between the field initialisations, similar to how it keeps them in struct declarations.

Reply:
TLDR: this is the version of `./vlib/v/fmt/tests/struct_init_with_comments_keep.vv`, that I think should be kept the same, and currently it is not:
```v
module abcde

pub struct Builder {
pub mut:
	// inline before field
	buf       []u8
	str_calls int

	len          int
	initial_size int = 1
}

pub fn new_builder(initial_size int) Builder {
	return Builder{
		// buf: make(0, initial_size)
		buf: []u8{cap: initial_size}

		str_calls: 0 // after str_calls
		len:       0 // after len

		initial_size: initial_size // final
	}
}
```

Reply:
fixed.
```v 
module abcde

pub struct Builder {
pub mut:
	// inline before field
	buf          []u8
	str_calls    int
	len          int
	initial_size int = 1
}

pub fn new_builder(initial_size int) Builder {
	return Builder{
		// buf: make(0, initial_size)
		buf:       []u8{cap: initial_size}
		str_calls: 0 // after str_calls
		len:       0 // after len

		initial_size: initial_size // final
	}
}
```